app.js

```diff
@@ -458,41 +428,31 @@ io.use(function (socket, next) {
 io.sockets.on('connection', function (socket) {
     logger.info('openHAB-cloud: Incoming openHAB connection for uuid ' + socket.handshake.uuid);
     socket.join(socket.handshake.uuid);
-    // Remove openHAB from offline array if needed
-    delete offlineOpenhabs[socket.handshake.uuid];
     Openhab.findOne({
```

Comment:
This could replaced with upsert atomic operation?

Reply:
Nice find!  Yes i think we could in fact use this method, and as an extra bonus, save ourselves an extra mongo call as well. I eliminated another expensive write to an unused collection in this function, so combined with this, both could really have a positive impact on performance when we have to restart a container and thousands of openHABs try and reconnect.

Reply:
Done!

Reply:
👍